concepticondata/concepticon.tsv

```diff
@@ -2338,22 +2338,22 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2337	OF THAT KIND (SUCH)	Miscellaneous function words	In that way, such, etc.	Other	
 2338	HOW LONG	Time	Asking for the duration of an event.	Other	
 2339	NEED (SOMETHING)	Cognition	Feeling the desire or the urge for something.	Other	
-2340	SUNNY	The physical world	Full of sunshine (mostly referring to &quot;a sunny day&quot;).	Property	
-2341	CLOUDY	The physical world	Full of clouds (mostly referring to &quot;a cloudy day&quot;).	Property	
+2340	SUNNY	The physical world	“Full of sunshine (mostly referring to “a sunny day”).”	Property	
+2341	CLOUDY	The physical world	“Full of clouds (mostly referring to “a cloudy day”).”	Property	
```

Comment:
For crab you use "instead of “ for the inner quotes. And from what was discussed in the issue, I thought it's supposed to be "x" around, and ""x"" within?

Reply:
Yes, your right. I changed the quotes accordingly.